client/src/components/globals/Navbar/NewNavbar.jsx

```diff
@@ -18,7 +18,7 @@ export default function NewNavbar({ position }) {
     // const user = useUserAuth();
     // console.log("FORMBRIN", user);
     formbricks.init({
-      environmentId: import.meta.env.VITE_REACT_APP_FORMBRICKS_API_KEY,
+      environmentId: import.meta.env.VITE_REACT_APP_FORMBRICKS_ENV_ID,
```

Comment:
Please use the previous env variable only, so that we don't need to change it in production and other places

Reply:
done